src/NuGet.Core/NuGet.Packaging/PackageExtraction/PackageExtractionBehavior.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace NuGet.Packaging.PackageExtraction
+{
+    public static class PackageExtractionBehavior
+    {
+        private const string XmlDocFileSaveModeEnvironmentKey = "NUGET_XMLDOC_MODE";
+
+        static PackageExtractionBehavior()
+        {
+            var xmlDocFileMode = Environment.GetEnvironmentVariable(XmlDocFileSaveModeEnvironmentKey);
```

Comment:
I guess. Although it doesn't change what the value would be since it's reading an Environment variable.